lib/features/start_lesson/sentence_match/widgets/level_widget.dart

```diff
@@ -0,0 +1,79 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:percent_indicator/linear_percent_indicator.dart';
+
+class LevelWidget extends StatefulWidget {
+  const LevelWidget({super.key});
+
+  @override
+  State<LevelWidget> createState() => _LevelWidgetState();
+}
+
+class _LevelWidgetState extends State<LevelWidget> {
+  @override
+  Widget build(BuildContext context) {
+    return  Column(
+              children: [
+                Padding(
+                  padding: const EdgeInsets.only(top: 20),
+                  child: Row(
+                    children: [
+                      Expanded(
+                        child: LinearPercentIndicator(
+                          lineHeight: 12,
+                          width: MediaQuery.of(context).size.width * 2 / 3,
```

Comment:
do not use mediaquery instead of it use padding i mean remove width and give it padding from left and right sides